src/Bicep.Core/TypeSystem/Az/ResourceTypeProviderFactory.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Azure.Bicep.Types;
+using Azure.Bicep.Types.Az;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Features;
+using Bicep.Core.Modules;
+using Bicep.Core.Registry.Oci;
+using Bicep.Core.Semantics.Namespaces;
+using Bicep.Core.TypeSystem.Az;
+using Newtonsoft.Json;
+
+namespace Bicep.Core.TypeSystem
```

Comment:
Move the file to match this namespace

Reply:
There are a few other files in the same folder - please could you adjust all of them?